tests/src/test/scala/cats/tests/MonadRecInstancesTests.scala

```diff
@@ -1,19 +1,52 @@
 package cats
 package tests
 
-import cats.data.{OptionT, StateT, Xor, XorT}
+import cats.data.{
```

Comment:
Should this file be renamed?

Also can this be added as a law somewhere as opposed to a separate check?